pkg/util/lifted/pubkeypin/pubkeypin_test.go

```diff
@@ -160,7 +160,7 @@ func TestHash(t *testing.T) {
 	actualHash := Hash(testCert(t, testCertPEM))
 	if actualHash != expectedHash {
 		t.Errorf(
```

Comment:
I guess we don't need to change test files because the log file does not using klog.

Reply:
@Garrybest Thanks for pointing out that it has been revised